service/task_history.go

```diff
@@ -315,6 +315,13 @@ func (uis *UIServer) taskHistoryPickaxe(w http.ResponseWriter, r *http.Request)
 		return
 	}
 
+	for i := range last {
```

Comment:
you can use the second argument for range here (and in the other places)

Reply:
I'm mutating the tasks inside the slice in place, so I don't want to use the second argument of range, which creates a copy.